src/components/ContentHighlights/ContentHighlightArchivedAlert.jsx

```diff
@@ -0,0 +1,49 @@
+import { Alert } from '@edx/paragon';
+import { useContext, useState } from 'react';
+import { NEW_ARCHIVED_CONTENT_ALERT_DISMISSED_COOKIE_NAME } from './data/constants';
+import { EnterpriseAppContext } from '../EnterpriseApp/EnterpriseAppContextProvider';
+import { enterpriseCurationActions, isArchivedContent } from '../EnterpriseApp/data/enterpriseCurationReducer';
+
+const ContentHighlightArchivedAlert = () => {
+  const archivedContentCookies = {};
```

Comment:
nit: should this variable name be updated to reflect localStorage instead of cookies?